bin/MessagesBundle_nl.properties

```diff
@@ -53,14 +53,14 @@ Circ_opt = Opties
 Macro_origin = Bekijk macro oorsprong
 View = Bekijk
 Export_completed = Exporteren is voltooid.
-Export_error = Er zijn fouten gemaakt met exporteren: 
+Export_error = Er zijn fouten gemaakt met exporteren:
 PDF_Font_error = Somige lettertypes zijn niet geëxporteerd. Gebruik Helvetica, Courier, Times of Symbol.
 Illegal_filename = Onjuiste bestandsnaam.
 File_not_found = Bestand niet gevonden.
 Print_uncomplete = Printen is niet volbracht.
-Open_error = Fout openen bestand: 
+Open_error = Fout openen bestand:
```

Comment:
Should not be present a space at the end of the resource, just after the ":"?
It is present in the original resource and it is the last character of the line. Check that your editor does not remove it automatically.